hoodie-client/src/main/java/com/uber/hoodie/io/storage/HoodieStorageWriterFactory.java

```diff
@@ -41,7 +41,7 @@
       R extends IndexedRecord> HoodieStorageWriter<R> newParquetStorageWriter(String commitTime, Path path,
       HoodieWriteConfig config, Schema schema, HoodieTable hoodieTable) throws IOException {
     BloomFilter filter = new BloomFilter(config.getBloomFilterNumEntries(),
-        config.getBloomFilterFPP());
+        config.getBloomFilterFPP(), false);
```

Comment:
should n't we pass the config here as well?

Reply:
good catch

Reply:
@n3nash without the config here, actually we would not have written dynamic filters at all during the tests?

Reply:
I ran tests isolated only on the bloom filter and generated UUID's + random strings (to simulate non uuid based keys), the test don't go through this code path.